### PR TITLE
release: cut the 5.2.0 release

### PR DESCRIPTION
**Pull request by SvetoslavTsenov**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
+<a name="5.2.0"></a>
+# [5.2.0](https://github.com/NativeScript/NativeScript/compare/5.1.2...5.2.0) (2019-02-08)
+
+
+### Bug Fixes
+
+* call onNavigatingFrom event only if it exists ([#6773](https://github.com/NativeScript/NativeScript/issues/6773)) ([19dfd16](https://github.com/NativeScript/NativeScript/commit/19dfd16))
+* **android:** raise resume event on activity.onPostResume() ([#6766](https://github.com/NativeScript/NativeScript/issues/6766)) ([46c9de0](https://github.com/NativeScript/NativeScript/commit/46c9de0))
+* iOS tests ([4c15f71](https://github.com/NativeScript/NativeScript/commit/4c15f71))
```
Comment (MartoYankov) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+* **modal:** Fix crash if modal is destroyed before dismissed in Android ([#6723](https://github.com/NativeScript/NativeScript/issues/6723)) ([8a32102](https://github.com/NativeScript/NativeScript/commit/8a32102))
+* **tab-view:** remove onBackPressed override ([#6755](https://github.com/NativeScript/NativeScript/issues/6755)) ([984f162](https://github.com/NativeScript/NativeScript/commit/984f162))
+* **view-android:** clear dialog fragment when closing modal view ([#6852](https://github.com/NativeScript/NativeScript/issues/6852)) ([8c80044](https://github.com/NativeScript/NativeScript/commit/8c80044))
+
+
+### Features
+
+* add rebeccapurple to known css colors ([#6819](https://github.com/NativeScript/NativeScript/issues/6819)) ([672c821](https://github.com/NativeScript/NativeScript/commit/672c821))
+* OnDiscardedError typings and event ([#6777](https://github.com/NativeScript/NativeScript/issues/6777)) ([28db2af](https://github.com/NativeScript/NativeScript/commit/28db2af))
+* **action-bar:** flat mode breaks ios safe area ([#6862](https://github.com/NativeScript/NativeScript/issues/6862)) ([603c901](https://github.com/NativeScript/NativeScript/commit/603c901))
```
Comment (MartoYankov) [inline CHANGELOG.md]: This should be moved to fixes. My bad with the commit naming.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,37 @@
...
+# [5.2.0](https://github.com/NativeScript/NativeScript/compare/5.1.2...5.2.0) (2019-02-08)
+
+
+### Bug Fixes
+
+* call onNavigatingFrom event only if it exists ([#6773](https://github.com/NativeScript/NativeScript/issues/6773)) ([19dfd16](https://github.com/NativeScript/NativeScript/commit/19dfd16))
+* **android:** raise resume event on activity.onPostResume() ([#6766](https://github.com/NativeScript/NativeScript/issues/6766)) ([46c9de0](https://github.com/NativeScript/NativeScript/commit/46c9de0))
+* **android:** animator restore logic on simulated nav ([#6710](https://github.com/NativeScript/NativeScript/issues/6710)) ([c034d6e](https://github.com/NativeScript/NativeScript/commit/c034d6e))
+* **android:** failure saving state in mixed parent/nested frame nav ([#6719](https://github.com/NativeScript/NativeScript/issues/6719)) ([4dc35a5](https://github.com/NativeScript/NativeScript/commit/4dc35a5))
+* **Android:** a crash on application restart after livesync changes ([#6756](https://github.com/NativeScript/NativeScript/issues/6756)) ([1f1d722](https://github.com/NativeScript/NativeScript/commit/1f1d722))
```
Comment (vchimev) [inline CHANGELOG.md]:
```suggestion
* **android:** a crash on application restart after livesync changes ([#6756](https://github.com/NativeScript/NativeScript/issues/6756)) ([1f1d722](https://github.com/NativeScript/NativeScript/commit/1f1d722))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,37 @@
...
+* **android-login:** use passwordHint for password placeholder ([25f4b56](https://github.com/NativeScript/NativeScript/commit/25f4b56))
+* **ios:** native view frame optimizations in nested scenario ([#6809](https://github.com/NativeScript/NativeScript/issues/6809)) ([08acd84](https://github.com/NativeScript/NativeScript/commit/08acd84))
+* **ios:** textview content clipped on every other newline ([#6864](https://github.com/NativeScript/NativeScript/issues/6864)) ([23dc84d](https://github.com/NativeScript/NativeScript/commit/23dc84d))
+* **ios-frame:** tearDownUI and reset UINavigationController ([#6817](https://github.com/NativeScript/NativeScript/issues/6817)) ([cea97c6](https://github.com/NativeScript/NativeScript/commit/cea97c6))
+* **list-view-android:** app crashes on ListView item template change ([#6634](https://github.com/NativeScript/NativeScript/issues/6634)) ([2085d1e](https://github.com/NativeScript/NativeScript/commit/2085d1e))
+* **modal:** Fix crash if modal is destroyed before dismissed in Android ([#6723](https://github.com/NativeScript/NativeScript/issues/6723)) ([8a32102](https://github.com/NativeScript/NativeScript/commit/8a32102))
```
Comment (vchimev) [inline CHANGELOG.md]:
```suggestion
* **modal:** fix a crash if modal is destroyed before dismissed in Android ([#6723](https://github.com/NativeScript/NativeScript/issues/6723)) ([8a32102](https://github.com/NativeScript/NativeScript/commit/8a32102))
```

**Comment by SvetoslavTsenov** [comment]
test package_version#rc

**Pull request merged**